logbook-spring-boot-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookAutoConfiguration.java

```diff
@@ -398,7 +398,7 @@ static FilterRegistrationBean newFilter(final Filter filter, final String filter
     @ConditionalOnWebApplication(type = Type.SERVLET)
     @ConditionalOnClass({
             javax.servlet.Servlet.class,
-            org.zalando.logbook.servlet.javax.LogbookFilter.class
+            org.zalando.logbook.servlet.LogbookFilter.class
```

Comment:
javax package here should stay, it's meant for backward compatibility with spring boot 2. If your local build fails, first run `mvn package`, so that maven-shade-plugin can generate the javax classes.

Reply:
Fixed